src/main/webapp/app/pool-list/pool-list.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { PoolListService } from 'app/pool-list/pool-list.service';
+import { Observable } from 'rxjs';
+import { Pool } from 'app/pool/pool';
+
+@Component({
+    selector: 'jhi-pool-list',
+    templateUrl: './pool-list.component.html',
+    styles: []
+})
+export class PoolListComponent implements OnInit {
+    pools: Observable<Pool[]>;
```

Comment:
You don't need observable to make it work since angular will re-render component if a data binded to it will change.

Reply:
I'm not sure what you mean. `this.poolService.getPools();` returns observable and I decided that assigning it to variable  and then use async ngFor is better then calling service method in template. 
I suppose you don't want to have observable fields in component so I will change that to `subscribe()`.